src/main/java/io/SchoolManagementSystem/SchoolManager.java

```diff
@@ -0,0 +1,10 @@
+package io.SchoolManagementSystem;
+
+public class SchoolManager {
+
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
```

Comment:
We may need to specify a relationship between this class and the Department

Reply:
Added this relationship, where the school manager has an arrayList of Department.